src/utils/proofs_api/parameters.rs

```diff
@@ -0,0 +1,156 @@
+// Copyright 2019-2024 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+//! This module contains the logic for storing and verifying the proofs parameters.
+//!
+//! The parameters are fetched from the network and stored in the cache directory. The cache directory can be set
+//! using the [`PROOFS_PARAMETER_CACHE_ENV`] environment variable. If not set, the default directory is used.
+
+use std::{
+    fs::File as SyncFile,
+    io::{self, copy as sync_copy, BufReader as SyncBufReader},
+    path::{Path, PathBuf},
+};
+
+use ahash::HashMap;
+use anyhow::{bail, Context};
+use blake2b_simd::{Hash, State as Blake2b};
+use cid::Cid;
+use serde::{Deserialize, Serialize};
+use tracing::{debug, warn};
+
+use super::is_env_truthy;
+
+const PROOF_DIGEST_LEN: usize = 16;
+
+/// Environment variable that allows skipping checksum verification of the parameter files.
+const TRUST_PARAMS_ENV: &str = "TRUST_PARAMS";
```

Comment:
nonblocking: if this is dangerous, I'd prefer `FOREST_FORCE_TRUST_PARAMS` or `FOREST_UNSAFE_TRUST_PARAMS`

Reply:
Same is in other comment; I was a bit hesitant to change the variable names to avoid breaking changes, but probably better do it now than later.